common/protob/messages-tezos.proto

```diff
@@ -75,7 +76,7 @@ message TezosSignTx {
      * Structure representing information for reveal
      */
     message TezosRevealOp {
-        optional TezosContractID source = 1;
+        optional bytes source = 1;
```

Comment:
Please change also the field number if you change the type.

`optional bytes source = 7;`

Same for other changes below.

Reply:
Will do.